clients/client-python/gravitino/filesystem/gvfs_config.py

```diff
@@ -0,0 +1,15 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+
+class GVFSConfig:
+    CACHE_SIZE = "cache_size"
+    DEFAULT_CACHE_SIZE = 20
+
+    CACHE_EXPIRED_TIME = "cache_expired_time"
+    DEFAULT_CACHE_EXPIRED_TIME = 3600
+
+    AUTH_TYPE = "auth_type"
+    DEFAULT_AUTH_TYPE = "simple"
```

Comment:
I think we should document these configs.

Reply:
Yeah, I think we can review and merge this PR #3931 firstly, then I can add the new config docs.